packages/docusaurus-theme-classic/src/theme/BlogListPage/index.tsx

```diff
@@ -24,10 +24,10 @@ function BlogListPage(props: Props): JSX.Element {
   } = useDocusaurusContext();
   const isBlogOnlyMode = metadata.permalink === '/';
   const title = isBlogOnlyMode ? siteTitle : 'Blog';
-
+  const {blogDescription} = metadata;
   return (
-    <Layout title={title} description="Blog">
-      <div className="container margin-vert--lg">
+    <Layout title={title} description={blogDescription}>
+      <div className="container margin-vert--xl">
```

Comment:
```suggestion
      <div className="container margin-vert--lg">
```

It's not the purpose of this PR to change the design. I'm open to discussing this but in another PR.

I think it looks weird on mobile to have as much

Reply:
oh! This might be accidentally merged from a conflict, I will correct it.